test/cli.test.js

```diff
@@ -88,6 +88,42 @@ describe('CLI', () => {
     expect(() => JSON.parse(data)).not.toThrow();
   });
 
+  it('should be able to read template files with a UTF-8 BOM', async () => {
+    const utf8BOM = Buffer.from([0xEF, 0xBB, 0xBF]);
+
+    let files = [
+      'templates/alertRules.json',
+      'templates/autoscaleSettings.json',
+      'templates/servicePlan.json',
+      'templates/website.json'
+    ];
+
+    let mockFiles = new Map();
+
+    for (let file of files) {
```

Comment:
As file is not modified in the loop, can be const.

Reply:
Ditto.